dashboard/app/views/parent_mailer/parent_permission_confirmation.html.haml

```diff
@@ -0,0 +1,26 @@
+%p
+  Dear Parent/Guardian,
```

Comment:
Curious if these emails are getting translated or if we just provide them in English. Also curious if CPA requires communicating in other languages than English.

Reply:
MVP is English, with Spanish as a nice-to-have